docs/source/aryn_cloud/gentle_introduction.md

```diff
@@ -0,0 +1,220 @@
+## A Gentle Introduction to the Aryn Partitioning Service 
+You can use the Aryn Partitioning Service to easily chunk and extract data from complex PDFs. The Partitioning Service can extract paragraphs, tables and  images and returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment  PDF documents and extract a table and an image from those documents using the python aryn-sdk.
```

Comment:
double space at 'tables and  images' I think. 
Might rephrase that sentence to end with a gerundive/dependent clause rather than an independent clause, i.e. 'and returns' -> ', returning'. Just to avoid and _ and.

Reply:
ack